index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
- [ OPTIONAL],  You should connect your CSS to your HTML so that you styles can appear on your website. eg: 
![image](https://user-images.githubusercontent.com/43877689/170271752-383cafaa-78d6-469d-9965-17e7d6e1607f.png)

- [OPTIONAL ]  try to give you project a good title, so that it can look more profession, not  `<title>Document</title>`